### ISSUE TITLE
Cannot install Ambient Weather Station integration

### ISSUE THREAD

**Issue by sinemaker** [home-assistant/home-assistant.github.io#28348]
### Feedback

I was able to load this integration once and it worked for two days.  Then it stopped.  Since then I have been unable to get it to work.  I uninstalled everything, even went back in time to a HAOS backup.

When I try to install the integration it prompts me to enter my API Key and Application key.  I enter these and then it just hangs -- the circle keeps turning until I exit out.

Above I did not add these instructions to my configuration.yaml
ambient_station:
  api_key: “your_api_key”
  app_key: “your_application_key”

So I added these to my configuration.yaml.  Before Re-loading HA I check the configuration.yaml and this process fails, or rather, does not return until I exit it.
If I simply try to update all yaml configurations it fails -- clearly HA does not like these lines in my configuration.yaml
So I removed them.
Any assistance would be welcome.
John

### URL

https://www.home-assistant.io/integrations/ambient_station/

### Version

2023.7.3

**Comment by joostlek** [comment]
Could you maybe try removing the YAML from the configuration and adding this integration via the UI?

**Comment by sinemaker** [comment]
My problem is that I cannot add the Ambient Weather Station integration with the UI, even without adding the three outdated lines to the configuration.yaml.  I simply mentioned that I had also tried to add these lines, but concluded that HA didn't like them so I took them back out.

I guess I don't understand how to find the latest instructions for HA integration.  When I google "home assistant" and "Ambient Weather Station" I get this link that appears to be an official HA document.
https://www.home-assistant.io/integrations/ambient_station/
I gather from the response that somewhere in the 500 or so PR's that have subsequently been written on github (not sure what they are) there is one that says that yaml commands should not be used.  Is there a better or faster way to get to updated instructions?  When things change where do the corrected instructions get posted?

It is a sincere question because I obviously don't know and I think more experienced users do.

Bottom line, I still cannot add this integration to HA.

**Comment by joostlek** [comment]
2 things:

1. You caught us off guard by mentioning a mistake in our documentation which has been there since 2019 or so. Thanks for this. (I fixed it, hence this issue is automatically closed)

2. This repository is for issues and feedback on documentation. Please create a new issue over at GitHub.com/home-assistant/core and fill in the form and we'll take a look at it :)

**Comment by joostlek** [comment]
Oh, I just read it again and you don't have a direct issue (I skimmed the text and saw the 500 as an error code). If you check the page again it should be updated (or at least, the old stuff is removed)
